app/build.gradle

```diff
@@ -113,6 +113,9 @@ dependencies {
     // Used for base encoding and URI escaping
     implementation 'com.google.guava:guava:28.0-android'
 
+    // Used for Orbot Helper
+    implementation 'info.guardianproject.netcipher:netcipher:2.1.0'
```

Comment:
Since we only use 2 static/very basic methods of this library, do you think we can just copy them and remove the dependency?
And maybe if we need more functionality we can add it?

Reply:
Good point.